package.json

```diff
@@ -66,4 +66,4 @@
     "react-scripts": "3.4.3",
     "typescript": "3.7.5"
   }
-}
\ No newline at end of file
+}
```

Comment:
Please remove this. The version bump workflow will remove it anyway.

Reply:
Newline has been removed